### ISSUE TITLE
Display custom steps in a human-friendly way

### ISSUE THREAD

**Issue by aml2610** [codeceptjs/CodeceptJS#1534]
#### What are you trying to achieve?

I would like custom steps to be displayed in the same, human-friendly way as predefined steps.
e.g I should see:
```
successful login with email and password
    I try to log in with credentials 
       - I fill field "email-input", "email"
       - I fill field "password-input", "password"
```
where the first line is the scenario text, the second line is my custom step, specified by a function called `tryToLogInWithCredentials` on the actor, and the next two lines are smaller, low-level predefined steps specified by the `fillField` function on an actor.

#### What do you get instead?

```
successful login with email and password
    I: tryToLogInWithCredentials 
    I fill field "email-input", "email"
    I fill field "password-input", "password"
```

### Details

* CodeceptJS version: 2.0.6
* NodeJS Version: 8.11.0
* Operating System: 
* Protractor || WebDriverIO || Nightmare version (if related)
* Configuration file:

```js
exports.config = {
  tests: './tests/*_test.js',
  output: './output',
  helpers: {
    Puppeteer: {
      url: 'http://localhost'
    }
  },
  include: {
    I: './steps_file.js'
  },
  bootstrap: null,
  mocha: {},
  name: 'app'
}
```

Had a quick look at the source code, I think the `humanize` function from `step.js` could be used, I am happy to help with implementation if you decide this would be useful.

**Comment by PeterNgTr** [comment]
I guess you can create the PR then let‘s see how things are going:)

**Comment by aml2610** [comment]
OK I did some more digging and found that what I'm asking for was actually implemented in https://github.com/Codeception/CodeceptJS/pull/1094 but then in https://github.com/Codeception/CodeceptJS/pull/1212 some of the support was removed, namely:
- `MetaStep` got its own implementation of `humanize` which shadows the one on the base `Step` class (currently line 130 in `step.js`):
```
  humanize() {
    return this.name;
  }
```

Since `name` is just the name of the method e.g `logInWithCredentials` this is what causes custom-steps to be displayed as `I: loginWithCredentials` instead of `I: log in with credentials`.

- then a second change was on (currently) line 161 of `step.js`, replacing
`return new MetaStep(name, fnName);` with
`return new MetaStep(`${name}:`, fnName[2]);`

This causes it to display the column i.e `I: loginWithCredentials` instead of `I log in with credentials`

The PR that made those changes seems to be integrating Allure, I was wondering what was the specific reason behind making these 2 particular changes?
I tested briefly and if you revert them it will work as I would have expected it (I have not tested that nothing else breaks outside my own use case though)

Otherwise, if you run the test with the `--debug` flag you will indeed get the custom steps indented correctly and displayed in green colour which is nice so part of my issue is fixed by using this flag.
